src/index.css

```diff
@@ -1,27 +1,366 @@
+:root {
+  --logocolor: #0290ff;
+  --textcolor: #121212;
+  --secondcolor: #4386bf;
+  --progresscolor: #e8e8e8;
+}
+
 body {
-  margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  display: flex;
+  align-items: center;
+  margin: 200px 200px;
+  background-color: #fafafa;
+  padding: 20px;
+  box-shadow: 0 5px 8px 0 rgba(0, 0, 0, 0.2);
+  box-sizing: border-box;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  border-bottom: solid 1px #e8e8e8;
+  box-shadow: rgba(149, 157, 165, 0.2) 0 8px 24px;
+}
```

Comment:
- [ ] dear @Daniel-Kigozi, I notice your heading has no margin/padding from the right side and the navigation items are added to the bottom of the `Book Store` text. Please make sure to add the right color for the navigations, the appropriate spacing between elements and align all the elements inside the heading as shown in the [Zeplin](https://zeplin.io/) design.
- [ ] Also, There is too much padding from the top and left side for the display. Please consider reducing it. This is to make sure your app looks exactly like the given design.
- [ ] Besides the above points, I believe
### Figma design 
![Capturex](https://user-images.githubusercontent.com/69077061/218083200-d9ee2392-db60-4d3b-9103-978b6b8210a6.PNG)

### Your design.
 you missed the part where you add a user icon on the right corner of your header.
![Capture](https://user-images.githubusercontent.com/69077061/218083193-1d96d47a-f53f-4f8a-9b24-5ede2b751fd5.PNG)